c7n/resources/apigw.py

```diff
@@ -683,28 +684,34 @@ class WafV2Enabled(Filter):
     permissions = ('wafv2:ListWebACLs',)
 
     def process(self, resources, event=None):
-        target_acl = self.data.get('web-acl')
+        target_acl = self.data.get('web-acl', '')
         state = self.data.get('state', False)
-
         results = []
+
         wafs = self.manager.get_resource_manager('wafv2').resources(augment=False)
         waf_name_arn_map = {w['Name']: w['ARN'] for w in wafs}
-        target_acl_id = waf_name_arn_map.get(target_acl, target_acl)
+
```

Comment:
@kk1532 ,
Since [wafv2-enabled](https://github.com/cloud-custodian/cloud-custodian/blob/master/c7n/resources/appsync.py#L27-L76) and [set-wafv2](https://github.com/cloud-custodian/cloud-custodian/blob/master/c7n/resources/appsync.py#L79-L173) are common for cloudfront, appsync, can we make same logic? You can refer to [pr/7706 ](https://github.com/cloud-custodian/cloud-custodian/pull/7706/files?diff=split&w=0#diff-cd3604abbc9879f5dce359a15dc6476ad19b719c30d4e04128bc304afbb07bae)for cloudfront wafv2-enabled as well.

Reply:
`wafv2-enabled` filter has been updated with cloudfront logic.